pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -12,23 +12,19 @@ author: CrystalOnScript
 Use AMP's well-tested, cross-browser compatible and accessible components anywhere on the web with Bento AMP. Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP! You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Bento also lets you test out AMP components to see if the path to valid AMP is right for your use case.
 
 [tip type="important"]
- Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](https://blog.amp.dev/2021/01/27/bento/) for information on future plans and how to get involved! 
+Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](https://blog.amp.dev/2021/01/27/bento/) for information on future plans and how to get involved!
 [/tip]
 
-# Use Bento components in non-AMP pages
-
-Bento components are still in development and APIs might change. However, they're available for experimental use in a developer preview. This means you must enable the experimental flag either in the document or the console. This is no different from other [experiments in AMP](../learn/experimental.md). 
+Bento components are still in development and APIs might change. However, they're available for experimental use in a developer preview. This means you must enable the experimental flag either in the document or the console. This is no different from other [experiments in AMP](../learn/experimental.md).
 
 Keep an eye out for announcements and updates. While Bento AMP is in developer preview, the team will continue to make necessary changes to the API to improve features.
 
-
-## Enable Bento Experiment 
+## Enable Bento Experiment
```

Comment:
Should all these headings go up a level to `#` since the `#` header was removed?

Reply:
Imo now, otherwise each heading would turn into a `h1` which does not fit from a semantic perspective. Not sure how we handle this in other guides (//cc @CrystalOnScript )

Reply:
We should probably move the headers up a step (the side TOC UI depends on these headers) - or replace the removed h1. Maybe `# Get started` or `# Enable and use Bento`